dockerbuild/wait_for_mongodb.sh

```diff
@@ -8,13 +8,12 @@ set -e
 thrift_version="$1"
 host="$2"
 
-# 'foo' doesn't matter here. Just needs something so that $host is interpreted
-# as a hostname and not a database name.
-until mongo "$host"/foo --eval "db.getCollectionNames()" > /dev/null 2>&1; do
+export PASSWORD=$(< /mongo/mongodb-password)
```

Comment:
Normally (with docker-compose) you'd want to use an existing environment variable for a secret, not mount a secret.

This should probably be removed.

Reply:
commented out line 11 but I only have mongo working with k8s at the moment.